### PR TITLE
Added New Socket Events Usage

### PR DESCRIPTION
**Pull request by malikkulsoom**
- Added socket interfaces for sent for settlement and re-queue notifications

### CODE REVIEW

File: tests/fireflyClient.test.ts [hunk 1]
```diff
@@ -1009,6 +1010,28 @@ describe("FireflyClient", () => {
       });
     });
 
+    it("should receive an sent for settlement event when trade is performed", (done) => {
+      const callback = (update: OrderSettlementUpdateResponse) => {
+        console.log(update)
```
Comment (burhanuddin03) [inline tests/fireflyClient.test.ts:1015]: pls remove console.log

File: src/interfaces/routes.ts [hunk 2]
```diff
@@ -120,6 +120,23 @@ export interface PlaceOrderResponse extends OrderResponse {
   postOnly?: boolean;
 }
 
+export interface OrderSettlementUpdateResponse {
+  orderHash: string;
+  userAddress: string;
+  symbol: string;
+  message: string;
+  quantitySentForSettlement: string;
+  orderQuantity: string;
```
Comment (burhanuddin03) [inline src/interfaces/routes.ts:129]: I remember we  added `orderQuantity` in requeued payload as well ?

File: tests/fireflyClient.test.ts (hunk 1 above)
Comment (malikkulsoom) [reply tests/fireflyClient.test.ts]: done

File: src/interfaces/routes.ts (hunk 2 above)
Comment (malikkulsoom) [reply src/interfaces/routes.ts:129]: orderQuantity is the total quantity of the order, I have started sending it from OME but haven't catered on Dapi yet that's why didn't add that here.

**Review by burhanuddin03 — APPROVED** [review]

**Pull request merged**